exec/exec.go

```diff
@@ -79,16 +87,23 @@ func New() Interface {
 
 // Command is part of the Interface interface.
 func (executor *executor) Command(cmd string, args ...string) Cmd {
-	return (*cmdWrapper)(osexec.Command(cmd, args...))
+	return &cmdWrapper{osexec.Command(cmd, args...)}
 }
 
 // LookPath is part of the Interface interface
 func (executor *executor) LookPath(file string) (string, error) {
 	return osexec.LookPath(file)
 }
 
+// CommandContext is part of the Interface interface.
+func (executor *executor) CommandContext(ctx context.Context, cmd string, args ...string) Cmd {
+	return &cmdWrapper{osexec.CommandContext(ctx, cmd, args...)}
+}
+
 // Wraps exec.Cmd so we can capture errors.
-type cmdWrapper osexec.Cmd
+type cmdWrapper struct {
```

Comment:
Why? This was defined explicitly NOT to inherit methods, and to force us to declare methods we want to pass-through.

Reply:
because I feel this casting `(*osexec.Cmd)(cmd).CombinedOutput()` looks weird. if it's defined as embeded type then the casting can be avoided. it's basically the same as `ExitErrorWrapper` below. anyway, I don't have strong opinion on this. it could be as is.